frontend/src/components/main/FutureProjectionForm.tsx

```diff
@@ -1,38 +1,32 @@
 import React, { useState } from 'react';
 import { TextField, Button, Box, MenuItem, Select, InputLabel, FormControl } from '@mui/material';
 import { SelectChangeEvent } from '@mui/material/Select';
-import { FutureProjection, KeyFactor, Probability, ProjectionType } from '@/types';
+import { FutureProjection, Probability, ProjectionType, KeyFactor } from '@/types';
 import { format, parseISO } from 'date-fns';
```

Comment:
mach noch ein npm i date=fns

Reply:
Ist in der package.json
Sollte eigentlich funktionieren.